### PR TITLE
Issue #3383: Skip dry run build item for maven release commit

### PR DESCRIPTION
**Pull request by ahmadshabib**
Issue #3383 
@romani

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -254,21 +254,30 @@ script:
...
+    fi
+  - echo "SKIP_CI_FOR_FILES="$SKIP_CI_FOR_FILES
+  - echo "SKIP_CI_FOR_FILES="$SKIP_CI_FOR_COMMIT
   - |
     set -e
-    if [[ $SKIP_CI == 'false' ]];
+    if [[ $SKIP_CI_FOR_FILES == 'false' && $SKIP_CI_FOR_COMMIT == 'false']];
     then
-         eval $CMD;
+         eval $CMD
```
Comment (romani) [inline .travis.yml]: If this requirement ?

File: .travis.yml [hunk 2]
```diff
@@ -65,7 +65,7 @@ matrix:
       env:
         - DESC="Releasenotes generation"
         - CMD="./.ci/travis/travis.sh releasenotes-gen"
-        - SKIP_CI="false"
+        - SKIP_CI_FOR_FILES="false"
```
Comment (romani) [inline .travis.yml]: Please update name in all other CI configs. To keep name consistent.
https://github.com/checkstyle/checkstyle/search?utf8=%E2%9C%93&q=SKIP_CI&type=

**Comment by rnveach** [comment]
CI failure
>  eval: line 105: syntax error in conditional expression: unexpected token `;'

**Comment by romani** [comment]
@rnveach , what is better name SKIP_CI_BY_FILES or SKIP_CI_FOR_FILES ?

**Comment by rnveach** [comment]
> SKIP_CI_BY_FILES or SKIP_CI_FOR_FILES ?

It depends on context.
I would say `FOR_FILES` would be acceptable and make sense if the value was a list of files. Ex: `SKIP_CI_FOR_FILES = {"pom.xml", "README.md"}`
`BY_FILES` makes more sense for a boolean type, imo. I think the author was going with `FOR_FILES` to denote something like "skip CI because list of files matched ignore list" and kept trimming it down and came up with "FOR" for a short name.

In this situation `SKIP_CI_BY_FILES` makes more sense to me. I can't think of a better name right now.

**Comment by romani** [comment]
SKIP_CI_BY_FILES should be used. Second variable should be '_BY_' too.

**Comment by ahmadshabib** [comment]
> SKIP_CI_BY_FILES should be used. Second variable should be '_BY_' too.

On it :D

**Comment by ahmadshabib** [comment]
Hey @romani,  Does it need any further development ?

**Pull request merged**